spec/models/prison_spec.rb

```diff
@@ -91,6 +91,18 @@
         expect(described_class.nomis_ids).to match_array(['WSI'])
       end
     end
+
+    describe '.enabled_nomis_ids' do
+      before do
+        allow(described_class).to receive(:all).
+          and_return([double('prison', nomis_id: 'WSI').as_null_object,
```

Comment:
Suggestion: `object_double`, in order to ensure the public interface to the model is correct.

```ruby
object_double(described_class.new, nomis_id: 'WSI').as_null_object
```